transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/ExpireDateUtils.java

```diff
@@ -111,7 +115,7 @@ int getCursorPosition(int newLength, int editStart, int editAdd) {
     }
 
     @NonNull
-    String[] getDateParts(@NonNull @Size(max = 4) String input) {
+    String[] getDateParts(@NonNull @Size(max = 4) final String input) {
         String[] parts = new String[2];
         if (input.length() >= 2) {
             parts[0] = input.substring(0, 2);
```

Comment:
May i know why we are trusting string operations instead of predefined Date parsing classes(like Calender, Date, SimpleDateFormat etc). In my point of view more code, attention needed for date parsing using string operations.

Reply:
Please look at my response to the other comment